requirements.txt

```diff
@@ -0,0 +1,4 @@
+dbt-fal[postgres]
+dbt-postgres
+fal
+slack_sdk
```

Comment:
This dependency was missing from the instructions. Now even though it's automatically installed the `notify.py` script still won't work unless they setup the Slack credentials. I believe that is still a missing step in our Instructions. Any thoughts?

I think we could:

1. add a section on the readme;
2. potentially add a new script as an example that doesn't require any external credentials (alongside with the Slack one, not a replacement)

Reply:
that is a valid point, we can add it as a step in the readme. You can also update the `schema.yml` setup and the script to not read from env variables, but from `with` https://docs.fal.ai/fal/fal-dbt/structured-hooks